twisted/python/test/test_setup.py

```diff
@@ -49,11 +52,13 @@ def test_conditionalExtensions(self):
 
     def test_win32Definition(self):
         """
-        When building on Windows NT, the WIN32 macro will be defined as 1.
+        When building on Windows NT, the WIN32 macro will be defined as 1 on
+        the extensions.
         """
         ext = ConditionalExtension("whatever", ["whatever.c"],
                                    define_macros=[("whatever", 2)])
-        args = get_setup_args(conditionalExtensions=[ext])
+        self.patch(_setup, '_EXTENSIONS', [ext])
```

Comment:
Per https://github.com/twisted/twisted/pull/467#discussion_r74916242, this would not have a `patch` call...